shell/ci/testing/setup-devenv.sh

```diff
@@ -48,7 +49,18 @@ if [[ -n $CI ]]; then
     tempDir=$(mktemp -d)
     cp "$DIR/../../../.tool-versions" "$tempDir/" # Use the versions from devbase
     pushd "$tempDir" >/dev/null || exit 1
-    gh release -R getoutreach/devenv download --pattern "devenv_*_$(go env GOOS)_$(go env GOARCH).tar.gz"
+
+    # download the pre-release/latest version
+    REPO=getoutreach/devenv
+    if [[ $DEVENV_PRE_RELEASE == "true" ]]; then
+      echo "Runing pre-release"
+      TAG=$(gh release -R "$REPO" list | grep Pre-release | head -n1 | awk '{ print $1 }')
+    else
+      echo "Runing latest"
+      TAG=$(gh release -R "$REPO" list | grep Latest | awk '{ print $1 }')
```

Comment:
You can just omit `TAG` for latest, it defaults to the latest

Reply:
indeed... I added that just to be explicit .. so becomes easy while reading the code